dashboard/test/controllers/api/v1/sections_controller_test.rb

```diff
@@ -1022,7 +1022,7 @@ class Api::V1::SectionsControllerTest < ActionController::TestCase
 
     post :update, params: {
       id: section.id,
-      course_version_id: 1,
+      course_version_id: 2,
```

Comment:
please future proof this with something like `CourseVersion.maximum(:id).to_i + 1` instead of a hard-coded number.